src/java.base/share/classes/java/security/CodeSource.java

```diff
@@ -456,12 +457,16 @@ private boolean matchLocation(CodeSource that) {
                     return false;
                 }
                 if (this.sp == null) {
-                    this.sp = new SocketPermission(thisHost, "resolve");
+                    @SuppressWarnings("removal")
+                    var _ = this.sp = new SocketPermission(thisHost, "resolve");
```

Comment:
What's the reason for the `var _ =` bit ?

Reply:
`@SuppressWarnings` can not be applied to a an expression - it needs a (variable) declaration